livnote/frontend/desktop/components/notes/schema/editorSchema.ts

```
@@ -0,0 +1,328 @@
+    toDOM(node) {
+      const attrs: { [key: string]: any } = {};
+
+      if (node.attrs.align) {
+        attrs.style = `text-align: ${node.attrs.align}`;
+      }
+
+      if (node.attrs.indent && node.attrs.indent > 0) {
+        attrs["data-indent"] = node.attrs.indent;
+      }
+
+      return [`h${node.attrs.level}`, attrs, 0] as [string, Object, number];
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)